spec/support/saml_response_doc.rb

```diff
@@ -0,0 +1,264 @@
+require_relative 'saml_auth_helper'
```

Comment:
Add newline here.

I've not seen `require_relative` before.  Do we really need it here?